docs/data/base/components/button/UseButton.js

```diff
@@ -1,7 +1,8 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
 import clsx from 'clsx';
-import { useButton } from '@mui/base/ButtonUnstyled';
+
```

Comment:
Running `yarn docs:typescript:formatted` creates this space. I don't know why 😅

Reply:
This could be interesting to investigate and debug (not necessarily in this PR, though) 😁